test/hotspot/jtreg/compiler/vectorapi/TestVectorShuffleIota1024.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2021, Huawei Technologies Co. Ltd. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package compiler.vectorapi;
+
+import jdk.incubator.vector.ByteVector;
+import jdk.incubator.vector.VectorSpecies;
+import jdk.incubator.vector.VectorShuffle;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/*
+ * @test
+ * @bug 8266720
+ * @modules jdk.incubator.vector
+ * @run testng/othervm compiler.vectorapi.TestVectorShuffleIotaByte1024
```

Comment:
Perhaps the test can be annotated, declaring it should only execute on ARM/SVE platforms. See the use of the `@requires` clause used in other JDK tests.

Reply:
Thank you for you review. I think this test is used for any arch which has `ByteVector.SPECIES_MAX == 1024`.

Reply:
But we know which arches don't support, x86, PPC etc.

I am unsure why existing shuffle tests do not catch this problem. In fact i would prefer we focus on that if we can rather than adding a specific test. Would you mind looking to see if see if we can expand on the existing `shuffleTest`?

Reply:
* Q : Why existing shuffle tests do not catch this problem? 
* A: Because we need `vector_length >= 1024`. However, in x86 we don't have this env because the longest register of x86 is 512 in AVX512.